src/index.html

```diff
@@ -1,13 +1,43 @@
 <!doctype html>
+<%
+  const mapToHtml = (filteredChunks, chunks) => filteredChunks.map(chunk => `<script type="text/javascript" src="${chunks[chunk].entry}"></script>`).join('\n    ');
+
+  const headChunks = (headChunks, chunks) => {
+    if (!headChunks) {
+      return '';
+    }
+    return mapToHtml(headChunks, chunks);
+  };
+
+  const bodyChunks = (headChunks, chunks) => {
+    let body = Object.keys(chunks);
+
+    if (headChunks) {
+      body = body.filter(chunk => !headChunks.includes(chunk));
+    }
+
+    if (!body) {
+      return 'undefined';
+    }
+
+    return mapToHtml(body, chunks);
+  };
+%>
 <html lang="<%= htmlWebpackPlugin.options.lang %>" dir="<%= htmlWebpackPlugin.options.dir %>">
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <%= htmlWebpackPlugin.options.newRelicJS %>
+    <%= htmlWebpackPlugin.options.headHtml.join('\n    ') %>
+    <%= htmlWebpackPlugin.files.css.map(css =>`<link href="${css}" rel="stylesheet">`).join('\n    ') %>
+    <%= headChunks(htmlWebpackPlugin.options.headChunks, htmlWebpackPlugin.files.chunks) %>
+    <% if (htmlWebpackPlugin.files.favicon) { %>
+    <link rel="shortcut icon" href="<%= htmlWebpackPlugin.files.favicon%>">
+    <% } %>
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
-  <body style="height: 100%;">
+  <body>
     <div id="root"></div>
+    <%= bodyChunks(htmlWebpackPlugin.options.headChunks, htmlWebpackPlugin.files.chunks) %>
```

Comment:
I'm a little confused, does the headChunks config also allow someone to arbitrarily load bundles into the index.html as well? Should this be branded as `headChunks` or just `nonWebpackedChunks` (but better)

Reply:
head chunks would be webpacked. This is actually not used in this PR but will be used in a future PR. head chunks is internal only.

Reply:
Basically the goal of head and body chunks is to give us more control of where chunks are being placed. In a future enhancement there are some webpacked entry points that need to be placed in the head while still placing other chunks in the body.

Reply:
makes sense. the naming just seems a bit weird from site-config & the html option to then be sorted in the index.html